nixos/modules/services/torrent/opentracker.nix

```diff
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.opentracker;
+
+  out = {
+    options.services.opentracker = api;
+    config = lib.mkIf cfg.enable imp;
+  };
+
+  api = {
+    enable = mkEnableOption "opentracker";
+
+    package = mkOption {
+      type = types.package;
+      description = ''
+        opentracker package to use
+      '';
+      default = pkgs.opentracker;
+    };
+
+    extraOptions = mkOption {
+      type = types.separatedString " ";
+      description = ''
+        Configuration Arguments for opentracker
+        See https://erdgeist.org/arts/software/opentracker/ for all params
+      '';
+      default = "";
+    };
+
+    user = mkOption {
+      type = types.str;
+      description = ''
+        User which will run opentracker. by default opentracker drops all
+        privileges and runs in chroot after starting up as root.
+      '';
+      default = "opentracker";
+    };
+
+    workDir = mkOption {
+      type = types.path;
+      description = ''
+        Working directory of opentracker
+      '';
+      default = "/var/empty";
```

Comment:
This default suggests to me that opentracker won't generate any data (it will fail/crash if it tries). Given this, why would I want to change the work directory?